cardano-db-sync/src/Cardano/DbSync/Default.hs

```diff
@@ -213,7 +257,7 @@ insertLedgerEvents syncEnv currentEpochNo@(EpochNo curEpoch) =
     handler ev =
       case ev of
         LedgerNewEpoch en ss -> do
-          lift $ do
+          lift $
```

Comment:
Was this change proposed by fourmolu? We usually avoid `do` when there is a single line afterwards

Reply:
ah no, I took them out my self